src/gui_tool/gui/sp_gui.py

```diff
@@ -115,34 +119,52 @@ def reset(self):
         self.section_statuses = [SectionStatus.ACTIVE]
 
     def split(self, loc):
```

Comment:
so the issue with having a `split` is it means that we always use the entire video right? What would someone do if they want to remove a section of the video because the video is too long or the section is an intro or something?

Reply:
If a section is set to SectionStatus.DELETED, there will not be a MetaDataItem for it passed on

For example, to remove frame 0-10
- go to 10
- make a split
- go to 9 ("press 'a')
- toggle the current section

Then only frames 11-end will remain